src/functions/purging_rmw.c

```diff
@@ -252,6 +252,11 @@ purge (const short purge_after, const struct waste_containers *waste, char *time
 
   short status = 0;
 
+  bool cmd_empty = 0;
+
+  if (getenv("RMWTRASH") != NULL)
```

Comment:
We must check to see if it's not NULL before trying to use it, otherwise we get a segfault.